docs/developer-docs/smart-contracts/write/resources.mdx

```diff
@@ -0,0 +1,75 @@
+---
+keywords: [beginner, write, resources, concept, ide, cdk]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { Tooltip } from "/src/components/Tooltip/Tooltip";
```

Comment:
I see Tooltip imported, but I don't see it used. Is that intentional?

Reply:
No, I forgot to insert the tooltip code. Thanks for catching that!